src/components/learner-credit-management/BudgetDetailRedemptions.jsx

```diff
@@ -3,16 +3,22 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import LearnerCreditAllocationTable from './LearnerCreditAllocationTable';
-import { useBudgetId, useOfferRedemptions } from './data';
+import { useBudgetId, useOfferRedemptions, useSubsidyAccessPolicy } from './data';
 
-const BudgetDetailRedemptions = ({ enterpriseUUID }) => {
+const BudgetDetailRedemptions = ({ enterpriseFeatures, enterpriseUUID }) => {
   const { enterpriseOfferId, subsidyAccessPolicyId } = useBudgetId();
+  const { data: subsidyAccessPolicy } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
   const {
     isLoading,
     offerRedemptions,
     fetchOfferRedemptions,
-  } = useOfferRedemptions(enterpriseUUID, enterpriseOfferId, subsidyAccessPolicyId);
-
+  } = useOfferRedemptions(
+    enterpriseUUID,
+    enterpriseOfferId,
+    subsidyAccessPolicyId,
+    enterpriseFeatures.topDownAssignmentRealTimeLcm,
+    subsidyAccessPolicy.subsidyUuid,
```

Comment:
[suggestion, non-blocking] If this component only relies on `subsidyAccessPolicy` just to pass it through to `useOfferRedemptions`, I'm wondering if it'd be a bit cleaner for the `useOfferRedemptions` hook itself to call `useSubsidyAccessPolicy` using the `subsidyAccessPolicyId` it's provided). It may help simplify tests too as this code path may need any mocks for `useSubsidyAccessPolicy`, provided `useOfferRedemptions` is mocked.